code/datums/mutations/hulk.dm

```diff
@@ -9,6 +9,9 @@
 	species_allowed = list("human") //no skeleton/lizard hulk
 	health_req = 25
 	instability = 40
+	var/scream_delay = 200
```

Comment:
@Ghommie This is 200 deciseconds

Reply:
What the heck, it was 20 last time I viewed it. Sorry, you are now right.

Reply:
Well I thought 20 secs would be fine but after testing that's is way to high a delay. I'll set it to five secs I guess